gnome-initial-setup/pages/account/gis-account-page.c

```diff
@@ -1403,6 +1403,9 @@ gis_account_page_init (GisAccountPage *page)
 void
 gis_prepare_account_page (GisDriver *driver)
 {
+  if (gis_driver_is_skipped_page (driver, PAGE_ID))
+    return;
```

Comment:
I re-checked the skip file approach and I actually like it more. I was a bit blinded with providing the list of skip files from the branding file itself, but we can actually ship a separate skip file using the logic that is already in place and that would remove all the other code that I added.